src/distrs/srdist.jl

```diff
@@ -0,0 +1,30 @@
+# functions related to studentized range distribution
+
+using ForwardDiff: derivative
+using QuadGK: quadgk
+
+import .RFunctions:
+    srdistcdf,
+    srdistccdf,
+    srdistlogcdf,
+    srdistlogccdf,
+    srdistinvcdf,
+    srdistinvccdf,
+    srdistinvlogcdf,
+    srdistinvlogccdf
+
+# pdf for numbers with generic types
+function srdistpdf(ν::Real, k::Real, x::Number)
+    Φ(x) = (1+erf(x / √2)) / 2
+    ϕ(x) = derivative(Φ, x)
```

Comment:
`normpdf`?

Reply:
No, I meant that if `Φ` (upper case) is the normal distribution CDF, then `ϕ` (lower case) is the normal distribution pdf.

Reply:
Ah yes, you're right.

Reply:
You can also get rid of the ForwardDiff.jl dependency